docker.go

```diff
@@ -164,6 +167,18 @@ func getRoleArnFromEnv(env []string) (role roleArn, policy string, err error) {
 			}
 		} else if v[0] == "IAM_POLICY" && len(v) > 1 {
 			policy = strings.TrimSpace(v[1])
+		} else if v[0] == "IAM_ROLE_NAME" && len(v) > 1 {
```

Comment:
It should not break anything since any full IAM role must start with "arn:iam:". So using the IAM_ROLE variable to accept just the role name will not clash with any valid existing use of the variable. It makes the usage less complex and error prone. For example, if you set IAM_ROLE and IAM_ROLE_NAME at the same time, knowing what happens is not entirely intuitive. In this case IAM_ROLE_NAME gets used unless AWS_ACCOUNT_ID is not set, then IAM_ROLE gets used. Using one variable removes a lot of edge cases.